src/chrome/content/rules/XFCE.org.xml

```diff
@@ -1,52 +1,53 @@
 <!--
-	Non-functional hosts
-		SSL peer certificate was not OK:
-			 bugs.xfce.org
-			 buildbot.xfce.org
-			 foundation.xfce.org
-			 i18n.xfce.org
-			 installit.xfce.org
-			 ns1.xfce.org
-			 ns2.xfce.org
-			 ns4.xfce.org
-			 pyxfce.xfce.org
-			 squeeze.xfce.org
-			 svn.xfce.org
-			 thunar.xfce.org
-			 translations.xfce.org
-			 www.tx-us.xfce.org
-			 www.us.xfce.org
-			 users.xfce.org
-			 www-test.xfce.org
-			 xfc.xfce.org
+	This domain contains a wildcard DNS record.
```

Comment:
Cannot reproduce

Reply:
Try something like https://this-is-a-wildcard-dns-record.xfce.org/ - it works (with a invalid certificate, anyway).